src/javascripts/components/blog/ArticleIndexPageNav.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+class ArticleIndexPageNav extends React.Component {
+
+  render() {
+    return (
+      <nav className="index-nav">
+        <ul>
+          <li>
+            <a href=""> <img src="./images/caret-left.svg" alt="Previous Page"/><span>Previous</span></a>
+          </li>
+          <li>
+            <a href=""><span>Next</span><img src="./images/caret-right.svg" alt="Next Page"/></a>
```

Comment:
you already have "Next" text, so you don't really need `alt="Next Page"`.  To a screen reader, that's going to read "Next Image:Next Page" which is redundant